test/verifiedTests/analysis/testSteadyCom/testSteadyCom.m

```diff
@@ -123,7 +127,7 @@
             cont = changeCobraSolver('ibm_cplex', 'LP');
         end
     else  % test any one of the other LP solvers
-        solverPrefer = {'gurobi'; 'glpk'; 'tomlab_cplex'; 'cplex_direct'; 'mosek'; 'dqqMinos'; 'quadMinos'};
+        solverPrefer = {'gurobi'; 'glpk'; 'tomlab_cplex'; 'cplex_direct'; 'mosek'}; %Minos cannot be run using parfor!
```

Comment:
Maybe add this note to the appropriate document: https://github.com/opencobra/cobratoolbox/blob/master/docs/source/notes/notes.md